testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -413,7 +413,7 @@ def __init__(self, name, enterprise=False):
         super().__init__(name, extra_files)
 
     def client_services(self):
-        services = self._docker_compose_cmd("ps --service").split()
+        services = self._docker_compose_cmd("ps --services").split()
```

Comment:
On my `ubuntu:22.04` container, I get this:

```
root@f43022ecc7a6:/# docker ps --services
unknown flag: --services
See 'docker ps --help'.
root@f43022ecc7a6:/# docker service ps
"docker service ps" requires at least 1 argument.
See 'docker service ps --help'.

Usage:  docker service ps [OPTIONS] SERVICE [SERVICE...]

List the tasks of one or more services
root@f43022ecc7a6:/# 
```

So I guess both are wrong..?

Reply:
Huh? I just got this from my Ubuntu 22 container...

Reply:
```
      root@6ef16a4d22ba:/# docker-compose ps --help
List containers.

Usage: ps [options] [--] [SERVICE...]

Options:
    -q, --quiet          Only display IDs
    --services           Display services
    --filter KEY=VAL     Filter services by a property
    -a, --all            Show all stopped containers (including those created by the run command)
```

Reply:
Ahh, ur using the `docker` binary, not `docker-compose`

Reply:
Stop confusing me more! @kacf

Reply:
Lol! Sorry, I didn't notice it either!